components/login.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../style.css">
+    <title>Shoemall | Login</title>
+</head>
+<body>
+
+    <nav class="nav">
```

Comment:
main navigation can be nested inside header

Reply:
if you saw my deploy preview i am giving width of 100% to nav and footer and for middle hero section I am giving 85%
so that's why put it outside